src/jacobis_method_iterations.cpp

```diff
@@ -0,0 +1,36 @@
+#include "../include/jacobis_method_iterations.hpp"
+
+std::vector<int> no_of_jacobi_iterations(std::vector<int> N_values_vector, double eps)
+{
+    int no_of_N_values = N_values_vector.size();
+    int N;
+    int n;
+    double h;
+    double a;
+    double d;
+    arma::mat A;
+    arma::vec eigenvalues;
+    arma::mat eigenvectors;
+    int maxiter;
+    int iterations;
+    bool converged;
```

Comment:
```suggestion
    int N, n;
    double h, a, d;
    arma::mat A;
    arma::vec eigenvalues;
    arma::mat eigenvectors;
    int maxiter, iterations;
    bool converged;
```
Legger sammen noen relaterte deklarasjoner på samme linje